draft-ietf-tls-tls13.md

```diff
@@ -2941,9 +2962,14 @@ the client and servers MUST supply "key_share" values as described
 in {{key-share}}.
 {:br}
 
+Any future values that are allocated must ensure that the transmitted
+protocol messages unambiguously identify which mode was selected by
+the server; at present, this is indicated by the presence of the "key_share"
+in the ServerHello.
+
 ### Early Data Indication
 
-When a PSK is used, the client can send application data
+When a PSK is used and early data is allowed for that PSK, the client can send application data
```

Comment:
This is somewhat awkward language, but I do think we want to note that not all PSKs allow this.

Reply:
+1